fetch_raw_data.py

```diff
@@ -156,6 +156,15 @@ def fetch_from_recovery_csv(user, google_cloud_credentials_file_path, raw_data_d
             TracedDataJsonIO.export_traced_data_iterable_to_jsonl(traced_runs, f)
         log.info(f"Exported TracedData")
 
+def fetch_beneficiary_file_csvs(google_cloud_credentials_file_path, pipeline_configuration, raw_data_dir):
+    for beneficiary_file_url in pipeline_configuration.beneficiary_file_urls:
+        beneficiary_file = beneficiary_file_url.split("/")[-1]
+
+        log.info(f"Saving '{beneficiary_file}' to file '{raw_data_dir}'...")
+        with open(f'{raw_data_dir}/{beneficiary_file}', "wb") as beneficiary_output_file:
```

Comment:
If these files don't change, consider doing this only if the file doesn't exist to improve performance and reduce costs.

Reply:
updated to download on if the file does not exist.